runtime/dynlink.c

```diff
@@ -211,15 +216,11 @@ void caml_build_primitive_table_builtin(void)
 {
```

Comment:
This code path doesn't initialize `symb_section` and `crcs_section`. It's not obvious, but this is OK.